mlir/lib/Dialect/Rock/Transforms/AccelEmitter.cpp

```diff
@@ -424,12 +465,14 @@ ArrayAttr WmmaEmitter::computeOutputTransforms(PatternRewriter &b, Location loc,
   // tile. For workitems 0 to 15 m_tid is 0 and they write to 0,2,4,...,14.  For
   // workitems 16 to 31 m_tid is 1 and they write at positions 1,3,5,..,15 .
   // This is outlined in https://gpuopen.com/learn/wmma_on_rdna3/
-  toMatrixC.embed(
-      "gemmM", 1, matrixM, {"m", "wave_m", "m_tid", "rep_i", "item_i"},
-      {mPerBlock, mPerWave, 1, wmmaInsn.inputLen, wmmaInsn.outputStride});
+  toMatrixC.embed("gemmM", 1, matrixM,
+                  {"m", "wave_m", "m_tid", "rep_i", "item_i"},
+                  {mPerBlock, wmmaInsn.inputLen, 1, mWaves * wmmaInsn.inputLen,
```

Comment:
Similarly, let's fix or standardize on the repeat names?

Reply:
Same here, I think that `rep_i` is fine, it's just that the stride is larger (before it was `inputLen`, now it is `inputLen*waves`